src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -236,5 +186,58 @@ private static void PublishFiles(IEnumerable<LibraryAsset> files, string outputP
                 File.Copy(file.ResolvedPath, Path.Combine(outputPath, Path.GetFileName(file.ResolvedPath)), overwrite: true);
             }
         }
+
+        public static int Main(string[] args)
```

Comment:
It would be good to keep the style consistent between our commands. If you're going to move Main to the bottom, do it for all of the commands.